dbt/adapters/athena/impl.py

```diff
@@ -195,12 +197,13 @@ def _parse_s3_path(s3_path: str) -> Tuple[str, str]:
         prefix = o.path.lstrip("/").rstrip("/") + "/"
         return bucket_name, prefix
 
-    @staticmethod
```

Comment:
Why there was a staticmethod before ? 🤔

Reply:
because self was not passed and everything was passed to the function, including the client....
as now we do `conn = self.connections.get_thread_connection()` can't be anymore static as we need self.